on-boarding/task-2/main.dart

```diff
@@ -89,8 +107,14 @@ void main() {
         break;
 
       case 6:
-        print('Enter task title of task you want to delete:');
-        String title = stdin.readLineSync() ?? '';
+        String title = '';
+        while (title.isEmpty) {
+          print('Enter task title of task you want to delete:');
+          title = stdin.readLineSync() ?? '';
+          if (title.isEmpty) {
+            print('Task title cannot be empty. Please try again.');
+          }
+        }
 
         Task? task = taskManager.searchTask(title);
```

Comment:
Typo (line 123):
In the 'Delete task' case, there is a typo in the error message. Instead of "Task Noft Found," it should be "Task Not Found."